lib/components/Announcement/Announcement.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
```

Comment:
Should this file be moved out of the top level components dir since it's not public anymore?